File: Downloads/Documents/S'K update/FindAbsoluteMinimum.js
array = [1,3,5,4,8,2,4,3,6,5,9]
x = 3
y = 2
firstNumber = 0;
secondNumber = 0;
let result = 0;

for(counter = 0; counter < array.length; counter++){
	if(x == array[counter]){
		firstNumber = counter;
	}
	if(y == array[counter]){
		secondNumber = counter;
	}
		result = Math.abs(firstNumber - secondNumber);
}
		console.log(result);

File: Downloads/Documents/S'K update/FindAbsoluteMinimum.java
import java.util.Arrays;
public class FindAbsoluteMinimum {
	public  int findAbsolute(int[] collectInput, int firstNumber, int secondNumber) {
		
		int movedNumber = 0;
		int number = 0;		
		int absolute = 0;
		
		for(int counter = 0; counter < collectInput.length; counter++){
				if(firstNumber == collectInput[counter] ){
					 movedNumber = counter;	
				}
				if(secondNumber == collectInput[counter]){
					 number= counter;
				}
				absolute = Math.abs(movedNumber - number);

				if(Math.abs(movedNumber - number)<absolute){
					absolute = Math.abs(movedNumber - number);
				}
				
			}
			 	return absolute;
	}				
}		